src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java

```diff
@@ -367,6 +371,15 @@ public void updateUser(UserImpl userObj, DataObject user)
                     jda, responseNumber,
                     userObj, oldAvatar));
         }
+        
+        if (!Objects.equals(oldFlags, newFlags))
```

Comment:
Why not just simple `!=`? This is less readable

Reply:
Because the above one also uses Objects.equals, so I do this here too for consistency.

Reply:
We use `Objects.equals` for things that can be null, int cannot be null so you should use `!=`.